src/gausskernel/storage/access/k2/pg_session.cpp

```diff
@@ -58,36 +57,38 @@ std::shared_ptr<PgTableDesc> PgSession::LoadTable(const PgOid database_oid, cons
 }
 
 std::shared_ptr<PgTableDesc> PgSession::LoadTable(const PgObjectId& table_object_id) {
-    std::string t_table_uuid = table_object_id.GetTableUuid();
-    std::shared_ptr<TableInfo> table;
+    std::string t_table_uuid = std::move(table_object_id.GetTableUuid());
```

Comment:
no need for move - it is already an rvalue

Reply:
fixed